instructor_admin/ui/Text/Translations/TextWord.elm

```diff
@@ -16,49 +36,40 @@ type TextWord
 
 
 textWordToString : TextWord -> String
-textWordToString text_word =
+textWordToString textWord =
     "("
         ++ String.join " "
-            [ toString (id text_word)
-            , toString (sectionNumber text_word)
-            , toString (instance text_word)
-            , toString (phrase text_word)
-            , toString (wordKindToGroup (wordKind text_word))
+            [ String.fromInt (id textWord)
+            , String.fromInt (sectionNumber textWord)
+            , String.fromInt (instance textWord)
+            , phrase textWord
+            , toString (wordKindToGroup (wordKind textWord))
             ]
         ++ ")"
```

Comment:
`textWordToString` is safe to delete.  The `toString` here is trying to convert a `Maybe TextGroupDetails` to a string, this was probably #a debugging function left over from initial dev.

Reply:
Sounds good, deleting it.